### PR TITLE
feat(DCOS-16435): Add package disclaimer

### PR DESCRIPTION
**Pull request by weblancaster**
**DCOS-16435** This PR is only for the disclaimer.

Because the button configuration new design isn't implemented yet the button looks a little off when on small screens.

**before**
screen shot 2017-06-27 at 4 50 09 pm

**after**
screen shot 2017-06-27 at 4 50 33 pm

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by nLight** [comment]
The link leads to 404 is it expected behaviour?

**Comment by weblancaster** [comment]
@nLight Looks like they are changing that because yesterday was working fine. If you go to the Jira ticket we can see the links asked to be added here are the same and 404.

I will reach out to see if they are changing the links but that should't block you from reviewing the rest/functionality. Thanks

File: src/js/pages/catalog/PackageDetailTab.js [hunk 1]
```diff
@@ -239,6 +239,16 @@ class PackageDetailTab extends mixin(StoreMixin) {
     );
   }
 
+  getTermsConditionUrl() {
+    const cosmosPackage = CosmosPackagesStore.getPackageDetails();
+
+    if (cosmosPackage.selected) {
```
Comment (ahoskins) [inline src/js/pages/catalog/PackageDetailTab.js]: Replace with `cosmosPackage.isCertified()`? Looks like that is used elsewhere in this file, and is the more readable option.

File: src/js/pages/catalog/PackageDetailTab.js [hunk 2]
```diff
@@ -310,8 +320,18 @@ class PackageDetailTab extends mixin(StoreMixin) {
                 </h1>
                 <p>{this.getPackageBadge(cosmosPackage, version)}</p>
               </div>
-              <div className="media-object-item">
+              <div className="media-object-item package-action-buttons">
                 {this.getInstallButtons(cosmosPackage)}
+                <p>
+                  By Deploying, you agree to {" "}
```
Comment (ahoskins) [inline src/js/pages/catalog/PackageDetailTab.js]: I don't think "Deploying" should be capitalized.

Comment (weblancaster) [inline src/js/pages/catalog/PackageDetailTab.js]: This sentence came from the document attached to the Jira @ahoskins

Comment (ahoskins) [inline src/js/pages/catalog/PackageDetailTab.js]: Interesting, fair enough.

File: src/js/pages/catalog/PackageDetailTab.js (hunk 1 above)
Comment (weblancaster) [inline src/js/pages/catalog/PackageDetailTab.js]: Good catch. It's addressed thanks.

File: src/js/pages/catalog/PackageDetailTab.js (hunk 2 above)
Comment (MatApple) [inline src/js/pages/catalog/PackageDetailTab.js]: @leemunroe shouldn't we lowercase "Deploying" - the current design in uppercased.

**Comment by MatApple** [comment]
@weblancaster can you please sit with @leemunroe tomorrow morning to verify behavior at various screen sizes and lowercasing of "Deploying". Thank you 💯

File: src/js/pages/catalog/PackageDetailTab.js (hunk 2 above)
Comment (nLight) [inline src/js/pages/catalog/PackageDetailTab.js]: Are we going to i18n this?

**Comment by leemunroe** [comment]
Here are the mockups for reference:
image
screen shot 2017-06-20 at 9 09 24 pm

Some things I'm noticing:

* Change the "Configure" button style to outline
* Change the "Deploy" button to primary (purple) rather than green (this is something we're trying to gradually roll out so we're not using "Success" buttons)
* Change "Deploying" to "deploying" (lower case d)
* The text "By deploying you agree..." is ideally vertically aligned with the "Certified 3.1x" text and be the same size and color as the version text
* The link for "terms & conditions" is ideally the same grey text so it isn't drawing too much attention
* Do you have a mobile screenshot? Not overly concerned about the mobile styles and alignments as long as the links and buttons are accessible

**Comment by weblancaster** [comment]
@leemunroe @MatApple the PR updated to follow the new style changes.

**desktop**
screen shot 2017-06-29 at 11 17 34 am

**mobile**
screen shot 2017-06-29 at 11 17 51 am

**Comment by leemunroe** [comment]
Beautiful 🎨

Ship it 🚢

File: src/js/pages/catalog/PackageDetailTab.js (hunk 2 above)
Comment (MatApple) [inline src/js/pages/catalog/PackageDetailTab.js]: @nLight I think we should hold off doing any more i18n for now.

File: src/styles/views/packages/styles.less [hunk 3]
```diff
@@ -1,5 +1,20 @@
 & when (@packages-view-enabled) {
 
+  .package-action-buttons {
+    text-align: left;
+
+    & a {
+      color: #848995;
```
Comment (MatApple) [inline src/styles/views/packages/styles.less]: Please don't use custom colors, instead we should use the variables 👍

Comment (MatApple) [inline src/styles/views/packages/styles.less]: I pushed a commit for you :)

File: src/styles/views/packages/styles.less [hunk 4]
```diff
@@ -3,8 +3,8 @@
   .package-action-buttons {
     text-align: left;
 
-    & a {
-      color: #848995;
+    a {
+      color: @small-color;
```
Comment (weblancaster) [inline src/styles/views/packages/styles.less]: awesome thanks.

**Comment by weblancaster** [comment]
retest please

**Comment by weblancaster** [comment]
retest this please

**Pull request merged**